projects/azure/README.md

```diff
@@ -240,6 +247,22 @@ source ./cluster/test-deploy/azure/secret
 oc create secret generic cluster-secrets-azure-env --from-literal=azure_client_id=${AZURE_CLIENT_ID} --from-literal=azure_client_secret=${AZURE_CLIENT_SECRET} --from-literal=azure_tenant_id=${AZURE_TENANT_ID} --from-literal=azure_subscription_id=${AZURE_SUBSCRIPTION_ID} -o yaml --dry-run | oc apply -n azure -f -
 ```
 
+## codecov-token
+
+This secret is used for pushing reports to [Codecov](https://codecov.io/)
+
+To rotate this secret:
```

Comment:
I guess the first step here is to get into the codecov console and generate a new token but from a superficial look I couldn't find how to do it. Any ideas?